src/components/assignment-item.tsx

```
@@ -5,15 +5,15 @@ import { useQuery } from '@tanstack/react-query'
 import { LocationObjectCoords } from 'expo-location'
 import { Dimensions, Image, Pressable, Text, View } from 'react-native'
 
+import { Map } from '@/interfaces'
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)